docs/java/tips-and-tricks/common-problems.md

```diff
@@ -0,0 +1,60 @@
+---
+id: sdk-java-common-problems
+title: Some common problems and how to address them
+hide_title: true
+hide_table_of_contents: false
+sidebar_label: Frequently Debugged Problems
+description: This article is a collection of frequently occurring symptoms and a short guidance on how to address them.
+keywords:
+- sap
+- cloud
+- sdk
+- debugging
+- FAQ
+- common mistakes
+- problems
+image:
+---
+
+### Tenant or Principal not available
+
+**Symptom:** Tenant or principal information is not available or an incorrect tenant is used.
+
+Possible causes:
+
+- A subscriber based JWT is passed in but it's validation fails, causing the SDK to fall back to the provider account.
```

Comment:
```suggestion
- A subscriber based JWT is passed in but its validation fails, causing the SDK to fall back to the provider account.
```

Reply:
Do we have a link to documentation that explains those terms (provider, subscriber)?

Reply:
Not yet, but that would fit well in the Multitenancy section we already have. @artemkovalyov Do you want to create a BLI?